File: Level1/11.문자열내p와y의개수/11.문자열내p와y의개수.py
def solution(s):
    answer = True
    numP = 0
    numY = 0
    for i in range(len(s)):
        if s[i]=='p' or s[i] == 'P':
            numP += 1
        elif s[i]=='y' or s[i] == 'Y':
            numY += 1
    
    if(numP!=numY):
        answer = False
    

    return answer

File: Level1/11.문자열내p와y의개수/11.문자열내p와y의개수.cpp
#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{
    bool answer = true;
    int numP = 0;
    int numY = 0;
    for(int i = 0 ; i<s.size() ; i++){
        if(s[i]=='p' || s[i]=='P'){
            numP++;
        }else if(s[i]=='y' || s[i]=='Y'){
            numY++;
        }
    }
    if(numP!=numY){
        answer = false;
    }

    return answer;
}